### DISCUSSION TITLE
API to verify user by password and username

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4895] [Q&A]
Hello There,

Is there any API(which can be called from any server) through which we can validate the user by username and password?

**Comment by ossdhaval** [comment]
@dhanan-np Just to understand more: Where are you storing user credentials? On Janssen Server or an external IDP like Active Directory?

**Reply by dhanan-np** [reply]
Hello @ossdhaval , Credentials are not stored on any of the application server.
Actually credentials will be entered by user on our application(Mobile Application) UI. We will make an api call to JansAuth from our backend server at that time. So basically we are not redirecting user to the (Jans) Auth-Server for user to enter the name and password.

**Comment by ossdhaval** [comment]
I am closing this discussion since it has been inactive for a long time. It can be reopened if needed.
